purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterDisplayMode.kt

```diff
@@ -1,16 +1,14 @@
 package com.revenuecat.purchases.customercenter.events
 
 import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
-import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 
 /**
  * Display mode for the Customer Center. Meant for RevenueCatUI use.
  */
 @Serializable
 @ExperimentalPreviewRevenueCatPurchasesAPI
-enum class CustomerCenterDisplayMode {
+enum class CustomerCenterDisplayMode(val value: String) {
 
-    @SerialName("full_screen")
-    FULL_SCREEN,
+    FULL_SCREEN(value = "full_screen"),
```

Comment:
I think the value is not used at all?

Reply:
It's related to https://github.com/RevenueCat/purchases-android/pull/2117#discussion_r1943160575

Reply:
Using SerialName now 🚀